### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by JaroCamphuijsen** [usgs-coupled/phreeqc3#89]
I see there is no license in this repository. Without an open source software license it is not clear whether and how this software can be used and adapted by others. A good place to find the right license with the right terms is here: https://choosealicense.com/

**Comment by BEEugene** [comment]
Agreed, only a disclaimer about no warranty can be found. Is this MIT licensed?

**Comment by greenmmq** [comment]
on the USGS website there is a User Rights Notice:

https://[water.usgs.gov/water-resources/software/PHREEQC/Phreeqc_UserRightsNotice.txt](https://water.usgs.gov/water-resources/software/PHREEQC/Phreeqc_UserRightsNotice.txt)

And on the related PHREEQCRM repo there is a CC0 1.0 license: 

https://github.com/usgs-coupled/phreeqcrm/blob/master/LICENSE

So perhaps it is as easy as just adding one or both of these to the repo. 

But per the point made by @JaroCamphuijsen, [No License](https://choosealicense.com/no-permission/) does create some legal ambiguity so it would ease minds to have this. 

@dlparkhurst @scharlton2
